### ISSUE TITLE
Update icon sizes and tab coloring

### ISSUE THREAD

**Issue by jnweiger** [owncloud/client#8037]
![image](https://user-images.githubusercontent.com/1108546/90804503-9bc11a80-e31a-11ea-9fc5-d86ce62a5e8d.png)

Shown changes:
* The highligted tab ("Activity") was a strong blue. This was too dominant with the now more decent icons.
* The tab is now more connected to the panel below.
* The bell icons are much smaller than before. Now same size as the icons in the tabs.
* The user icon in the list is a tad smaller, to separate better from the same icon in adjacent lines.
* Bottom line of the scroll boxes much lighter than the other three, to indicate "something is incomplete here", an affordance for scrolling down.

Not shown:
* The alternating background style of the lower scrollbox is less intrusive than the dash-separation in the upper scroll box. Suggest to use alternating background for both.

Inspired by https://github.com/owncloud/client/pull/8036
